nototools/unicode_data.py

```diff
@@ -783,8 +783,7 @@ def _read_emoji_data(lines):
     if m.group(2):
       continue
 
-    # discourage lots of redundant copies of seq_type
-    seq_type = intern(m.group(3).strip().encode('ascii'))
```

Comment:
FWIW the old `intern` builtin was moved to `sys.intern`: https://docs.python.org/3/library/sys.html?highlight=sys.intern#sys.intern